be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/MeetingHandler.scala

```diff
@@ -1,22 +1,27 @@
 package ch.epfl.pop.pubsub.graph.handlers
 
 import ch.epfl.pop.model.network.JsonRpcRequest
+import ch.epfl.pop.model.objects.DbActorNAckException
 import ch.epfl.pop.pubsub.graph.{ErrorCodes, GraphMessage, PipelineError}
 
-import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await}
+import scala.util.{Failure, Success}
 
 case object MeetingHandler extends MessageHandler {
 
   def handleCreateMeeting(rpcMessage: JsonRpcRequest): GraphMessage = {
-    rpcMessage.getParamsChannel.decodeChannelLaoId match {
-      case Some(_) =>
-        val ask: Future[GraphMessage] = dbAskWritePropagate(rpcMessage)
-        Await.result(ask, duration)
-      case _ => Right(PipelineError(
-          ErrorCodes.INVALID_DATA.id,
-          s"Unable to create meeting: invalid encoded laoId '${rpcMessage.getParamsChannel}'",
-          rpcMessage.id
```

Comment:
here as well, having this split on multiple lines was helpful to read it quickly.

Reply:
Understood, I agree that it is easily readable.